pkg/event/target/store.go

```diff
@@ -18,9 +18,18 @@ package target
 
 import (
 	"errors"
+	"fmt"
+	"strings"
+	"time"
+
 	"github.com/minio/minio/pkg/event"
 )
 
+const retryInterval = 3 // In Seconds
+
+// ErrNotConnected - indicates that the target connection is not active.
+var ErrNotConnected = errors.New("Not connected to target server/service")
```

Comment:
Overall review.

1. Capitalizing error messages are non-standard.
2. Do not export constants, variables, functions, methods and types if they are not used outside of a package.
3. Add unit than comment e.g. `const retryInterval = 3 * time.Second`

Reply:
Changed It :+1: